### ISSUE TITLE
How do you set the `page[size]` for the Lists.getListProfiles ?

### ISSUE THREAD

**Issue by bomosura** [klaviyo/klaviyo-api-node#12]

**Comment by bomosura** [comment]
I have tried both of this implementation, it stil lreturn the default 20 items per page
```
const profiles = await KlaviyoSdk.Lists.getListProfiles('myIdHere', {
    pageSize: 100
})
```


```
const profiles = await KlaviyoSdk.Lists.getListProfiles('myIdHere', {
    'page[size]': 100
  })

```

**Comment by Ian-Montgomery-Klaviyo** [comment]
After some investigation, this is a bug that you can't pass pageSize as an optional parameter. We will come out with a fix for this. 

You can always call the api directly (the page[size] parameter does work when passed directly to our api)

**Comment by Ian-Montgomery-Klaviyo** [comment]
Fixed in the new 2.1.1 release.
